training/nvidia-bootc/duplicated/ilab-wrapper/ilab

```diff
@@ -8,21 +8,54 @@ export ENTRYPOINT="/opt/python3.11/venv/bin/ilab"
 export PARAMS=("$@")
 
 for dir in "$HOME/.cache" "$HOME/.config" "$HOME/.local"; do
```

Comment:
Why is this duplicated?

Reply:
See https://github.com/containers/ai-lab-recipes/pull/708#issuecomment-2258421931